modules/PhyVersoBSP/phyverso_mcu_comms/evConfig.cpp

```diff
@@ -31,16 +32,21 @@ bool evConfig::open_file(std::string path) {
 
 // todo: needs to be refactored a lot
 bool evConfig::check_validity() {
-    std::cout << config_file.dump() << "\n\n";
+    std::cout << config_file.dump() << "\n\n"; // DEBUG
```

Comment:
Maybe left over from debugging?

Reply:
Yes, this was left over from debugging, the comment above the function was from pre config rework so I removed that as well in [44c90c1](https://github.com/EVerest/everest-core/pull/771/commits/44c90c1cac008ed939ff12ddb9f067ed3cb2612c).